settings.py

```diff
@@ -456,6 +456,10 @@
 ICDS_ENVS = ('icds',)
 UNLIMITED_RULE_RESTART_ENVS = ('echis', 'pna', 'swiss')
 
+# temporary for the transition to the new community / trial experience
+# so that PRs don't get too big
+ENABLE_NEW_TRIAL_EXPERIENCE = False
```

Comment:
you can basically ignore this as later this gets removed. It was a feeble attempt to split up the changes...

Reply:
In a PR this size, this sounds like it would have been nice to rebase away.